sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -199,3 +199,30 @@ ALTER TABLE `procedure_providers` ADD `type` VARCHAR(31) DEFAULT NULL;
 ALTER TABLE `procedure_answers` ADD `procedure_code` VARCHAR(31) DEFAULT NULL;
 #EndIf
 
+#IfNotRow users username oe-system
+INSERT INTO `users`(`username`,`password`,`lname`,`authorized`,`active`) VALUES ('oe-system','NoLogin','System Operation User',0,0);
+INSERT INTO `gacl_aro`(`id`, `section_value`, `value`, `order_value`, `name`, `hidden`)
+    SELECT max(`id`)+1,'users','oe-system',10,'System Operation User', 0 FROM `gacl_aro`;
+INSERT INTO `gacl_groups_aro_map`(`group_id`, `aro_id`)
+    VALUES (
+        (SELECT `id` FROM `gacl_aro_groups` WHERE parent_id=10 AND value='admin')
+        ,(SELECT `id` FROM `gacl_aro` WHERE `section_value` = 'users' AND `value` = 'oe-system')
+    );
+#EndIf
+
+#IfNotTable
+CREATE TABLE `export_job` (
+  `uuid` varchar(40) NOT NULL,
```

Comment:
TODO for me. revisit this after review the export_job mechanism.

Reply:
Would make uuid binary and a primary key (check out the `uuid_registry` `uuid` column for an example of this). And will show how to create a naturally ordered uuid below where the uuid is made (the smaller size of binary and the natural ordering make it as fast as possible for a uuid). The goal at some point is to use these uuid's instead of incrementing id's to help OpenEMR reach goal of offline support (ie. not always connected to server).

Reply:
Made this change and pushed it.